internal/cli/atlas/livemigrations/create.go

```diff
@@ -0,0 +1,183 @@
+	sourceProjectID             string
+	sourceSSL                   bool
+	sourceManagedAuthentication bool
+	sourceUsername              string
+	sourcePassword              string
+	store                       store.LiveMigrationCreator
+}
+
+func (opts *CreateOpts) initStore() error {
+	var err error
+	opts.store, err = store.New(store.AuthenticatedPreset(config.Default()))
+	return err
+}
+
+var createTemplate = `ID	PROJECT ID	SOURCE PROJECT ID	STATUS
+{{.ID}}	{{.GroupID}}	{{.SourceGroupID}}	{{.Status}}`
+
+func (opts *CreateOpts) Run() error {
+	if err := opts.askDestinationDropConfirm(); err != nil {
+		return err
+	}
+
+	if err := opts.askPassword(); err != nil {
+		return err
+	}
+
+	createRequest := opts.newCreateRequest()
+
+	r, err := opts.store.Create(opts.ConfigProjectID(), createRequest)
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *CreateOpts) newCreateRequest() *mongodbatlas.LiveMigration {
+	return &mongodbatlas.LiveMigration{
+		Source: &mongodbatlas.Source{
+			ClusterName:           opts.sourceClusterName,
+			GroupID:               opts.sourceProjectID,
+			Username:              opts.sourceUsername,
+			Password:              opts.sourcePassword,
+			SSL:                   &opts.sourceSSL,
+			CACertificatePath:     opts.sourceCACertificatePath,
+			ManagedAuthentication: &opts.sourceManagedAuthentication,
+		},
+		Destination: &mongodbatlas.Destination{
+			ClusterName: opts.destinationClusterName,
+			GroupID:     opts.ConfigProjectID(),
+		},
+		MigrationHosts: opts.migrationHosts,
+		DropEnabled:    &opts.destinationDropEnabled,
+	}
+}
+
+func (opts *CreateOpts) askDestinationDropConfirm() error {
+	if !opts.destinationDropEnabled {
+		return nil
+	}
+	confirmDrop := false
+	p := prompt.NewConfirm("Are you sure you want to drop the destination collections?")
+
+	if err := survey.AskOne(p, &confirmDrop); err != nil {
+		return err
+	}
+
+	if !confirmDrop {
+		return errors.New("user-aborted. Not dropping destination collections")
+	}
+	return nil
+}
+
+func (opts *CreateOpts) askPassword() error {
+	if opts.sourceManagedAuthentication {
+		return nil
+	}
+	if opts.sourcePassword != "" {
+		return nil
+	}
+	p := &survey.Password{
+		Message: "Password:",
+	}
+	if err := survey.AskOne(p, &opts.sourcePassword); err != nil {
+		return err
+	}
+	if opts.sourcePassword == "" {
+		return errors.New("no password provided")
+	}
+	return nil
+}
+
+func (opts *CreateOpts) validate() error {
+	if opts.sourceManagedAuthentication && opts.sourceUsername != "" {
+		return fmt.Errorf("--%s and --%s are exclusive", flag.LiveMigrationSourceManagedAuthentication, flag.LiveMigrationSourceUsername)
+	}
+	if !opts.sourceManagedAuthentication && opts.sourceUsername == "" {
+		return fmt.Errorf("MongoDB Automation is not managing authentication, --%s must be set", flag.LiveMigrationSourceUsername)
+	}
+	if opts.sourceCACertificatePath != "" {
+		opts.sourceSSL = true
+	}
+	return nil
+}
```

Comment:
Same as the validation point - we should do this in one place and reuse if we can.